src/renderer/subscribeToIPCAction.ts

```diff
@@ -0,0 +1,9 @@
+import { ipcRenderer } from 'electron'
+import { Action } from 'redux'
+import { IPCEvents } from 'src/constants'
+
+export const subscribeToIPCAction = (callback: (action: Action) => void): void => {
+    ipcRenderer.on(IPCEvents.ACTION, (_, action: Action) => {
```

Comment:
Same here, isn't the context bridge bi-directional? 🤔

Reply:
`contextBridge` isn't about communication itself (so doesn't imply direction of it) but only about _visibility_ or rather access to resources that usually run in restricted context - so without it, anything using `ipcRenderer` wouldn't be able to access it on renderer side. Hence why I'm exposing the parent functions in the preload script via the `contextBridge` so they can be used from the main execution context in renderer process

Reply:
Ah gotcha, makes sense